managed/services/telemetry/config.go

```diff
@@ -79,7 +79,7 @@ type DSGrafanaSqliteDB struct {
 }
 
 // DSConfigPMMDB telemetry config.
-type DSConfigPMMDB struct {
+type DSConfigPMMDB struct { //nolint:musttag
```

Comment:
wouldn't it be better to really tag fields?

Reply:
I don't know if it was made like this on purpose or not?

Reply:
@ritbl ^

Reply:
Ah, right. All untagged fields are part of structs that marked as for `no serialization`